prime-router/src/testIntegration/resources/datatests/translation-test-config.csv

```diff
@@ -1,6 +1,6 @@
 Input File,Input schema,Expected File,Output schema,Output format,Result
-CSV_to_HL7/sample-batch-pdi-20210608-0001.csv,primedatainput/pdi-covid-19,CSV_to_HL7/sample-batch-pdi-20210824-0001.hl7,fl/fl-covid-19,HL7_BATCH,PASS
-CSV_to_HL7/sample-single-pdi-20210608-0002.csv,primedatainput/pdi-covid-19,CSV_to_HL7/sample-single-pdi-20210824-0002.hl7,fl/fl-covid-19,HL7,PASS
+CSV_to_HL7/sample-batch-pdi-20210608-0001.csv,primedatainput/pdi-covid-19,CSV_to_HL7/sample-batch-pdi-20211108-0001.hl7,fl/fl-covid-19,HL7_BATCH,PASS
```

Comment:
These dates do not change.  The input and output filename are supposed to match, so we know which is which.